migrations/emailmonitor/oxa_email_monitor.py

```diff
@@ -0,0 +1,215 @@
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        c = str(row[3])
+        sql2 = "UPDATE oxa.oxa_activationsummary SET newaccount="+c+" \
+               WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+             
+        cursor.execute(sql2)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+# For each YYYY,MM,DD statistics raw in oxa.oxa_activationsummary table, this function updates the activated accounts statistics based on edxapp.auth_user table. This is rerunnable
+def update_accounts_activated():
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    sql = "SELECT YEAR(date_joined), MONTH(date_joined), DAY(date_joined), COUNT(*) FROM edxapp.auth_user \
+          WHERE is_active=1 \
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        c = str(row[3])
+        sql2 = "UPDATE oxa.oxa_activationsummary SET activated="+c+" \
+               WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+
+        cursor.execute(sql2)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+# For each YYYY,MM,DD statistics raw in oxa.oxa_activationsummary table, this function updates the non-activated accounts statistics based on edxapp.auth_user table. This is rerunnable
+def update_accounts_notactivated():
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    sql = "SELECT YEAR(date_joined), MONTH(date_joined), DAY(date_joined), COUNT(*) FROM edxapp.auth_user \
+          WHERE is_active=0 \
+          GROUP BY DATE(date_joined) \
+          ORDER BY DATE(date_joined)"
+
+    cursor.execute(sql)
+    results = cursor.fetchall()
+
+    for row in results:
+        y = str(row[0])
+        m = str(row[1])
+        d = str(row[2])
+        c = str(row[3])
+        sql2 = "UPDATE oxa.oxa_activationsummary SET notactivated="+c+" \
+               WHERE xyear="+y+" and xmonth="+m+" and xday="+d
+
+        cursor.execute(sql2)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+# For the given failed activation email line which has timestamp and user email address, update oxa database if it is not already processed
+# This is rerunnable for logs and it process a line only once in database
+def process_error_line( cursor, line ):
+
+    # From the activation email error line, with regular expression fetch IP, year, month, day, hour, minute, second, and email    
+    m = re.search('/tmp/oxa_log_files/(.+?)/.+:([0-9]{4}?)-([0-9]{2}?)-([0-9]{2}?)\s([0-9]{2}?):([0-9]{2}?):([0-9]{2}?),.+to\s"(.+?)"',line)
```

Comment:
Let's submit this first PR. Later we can improve it if want to invest more on this script.